### ISSUE TITLE
Change color button from grey to blue

### ISSUE THREAD

**Issue by Liubov-crypto** [aeternity/superhero-wallet#527]
In the settings, remove an account, I would suggest changing the colour of the buttons. Make the "cancel" button blue, and the "confirm" button grey. Thus, the focus shifts from deleting an account to cancelling.

![Знімок екрана 2020-09-22 о 1 29 01 пп](https://user-images.githubusercontent.com/69896204/93896061-25d51800-fcf9-11ea-8820-a100c630834d.png)

What do you think @onvisions ?

**Comment by mradkov** [comment]
I think otherwise. Buttons should be consistent everywhere.

If some change is needed I would vouch for lower opacity on the confirm button rather than switching colors.

**Comment by Liubov-crypto** [comment]
Ok. This is just my suggestion. Service providers always make a focus on desired user actions.

**Comment by mradkov** [comment]
@onvisions what do you think?

**Comment by onvisions** [comment]
1. @mradkov is right that we should keep the colors consistent. 
2. We can solve this issue that @Liubov-crypto raised with some better approach and a little bit more efforts.
I suggest the following:

- add link to text "backed up your seed phrase" (green, underlined) leading to Backup functionality;
- make the button Confirm disabled by default (40% opacity, cursor is arrow) and add a checkbox between the text and the buttons with the following text:

**I understand that this is a cryptocurrency wallet and I am responsible for backing up my account recovery seed phrase.**

When checkbox is checked the disabled Confirm button gets enabled. 

What do you think @mradkov and @Liubov-crypto ? I can make design for this if needed although we have already the same checkbox here:

![Component 23 – 25](https://user-images.githubusercontent.com/12225410/94935719-0e0a4a80-04d6-11eb-90df-4c9a0f99e04b.png)

**Comment by davidyuk** [comment]
I agree with the initial proposal, it is easy to implement and it looks consistent with user experience in other apps. For example, here is a remove dialog from macOS:

<img width="532" alt="Screenshot 2020-10-07 at 12 17 50" src="https://user-images.githubusercontent.com/9007851/95312104-2f34b780-0897-11eb-8249-7fa56e0a32a2.png">

**Comment by Liubov-crypto** [comment]
I don't know if it makes sense to send the user to the seed phrase after he/she presses the confirm button (delete account) and then back. Since the warning about deleting the account and backing up the seed phrase is already on the remove account page. But since I am not a designer, I will trust your decision @onvisions 

![Знімок екрана 2020-10-07 о 1 01 06 пп](https://user-images.githubusercontent.com/69896204/95317936-85592900-089e-11eb-9c2f-a525ff58687d.png)

**Comment by onvisions** [comment]
Something more on the topic to support my reasoning:
https://uxmovement.com/buttons/why-cancel-buttons-should-never-have-a-color/

**Comment by davidyuk** [comment]
Let's make this dialog according to it:
https://uxmovement.com/buttons/how-to-design-destructive-actions-that-prevent-data-loss/

**Comment by Liubov-crypto** [comment]
The old design. Can be closed.
